sdk/kubeflow/trainer/utils/utils.py

```diff
@@ -122,6 +122,25 @@ def get_resources_per_node(resources_per_node: dict) -> client.V1ResourceRequire
     return resources
 
 
+# TODO (andreyvelich): Change return type to IntOrString.
+def get_num_proc_per_node(resources_per_node: dict) -> object:
```

Comment:
@astefanutti I added this util func to get numProcPerNode based on CPU and GPU configuration, as we discussed before.
WDYT ?

Reply:
Sounds good, I agree with you, better integrate E2E asap, and we can work on #2407 separately.

Reply:
SGTM, we can construct Torch environment variables based on input numProcPerNode as we do in the current torch plugin.

Reply:
Sorry for your confusion. That is what I wanted to say.